test/cpp/jit/test_autodiff.cpp

```diff
@@ -81,6 +81,7 @@ variable_list grad(
       grad_outputs,
       true,
       false,
+      false,
```

Comment:
We might need to keep this since `execute()` still requires `accumulate_grad`. We've kept the changes in engine.h/engine.cpp as required for the Python api

Reply:
Ok